From 2a472c4fab230394842f252abd033d9d3c69f0c0 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Fri, 7 May 2021 14:39:53 -0400
Subject: [PATCH] Updated decorator requirement for #4718

incompatible with decorator 5.0.0 through 5.0.6, fix pinned version
by requiring >=5.0.7 (which we package) rather than 4.* which we don't

See https://github.com/networkx/networkx/pull/4773
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index eb1521d7..3cad13f3 100644
--- a/setup.py
+++ b/setup.py
@@ -124,7 +124,7 @@ package_data = {
     "networkx.utils": ["tests/*.py"],
 }
 
-install_requires = ["decorator>=4.3,<5"]
+install_requires = ["decorator>=5.0.7"]
 extras_require = {
     "all": [
         "numpy",
-- 
2.31.1

